### PR TITLE
feat: add logout screen design

### LINKED ISSUE #228: Missing Navigation in Profile Settings
Description: When in the profile settings page, in order to go back to the previous page, the user has to close the app and open it again, or sign out and sign in again.

Potential Solution: Add a back button to the profile settings page.

image

### PR DESCRIPTION
**Pull request by mhenriette**
Closes #228

### CODE REVIEW

File: components/cards/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { View, StyleSheet } from "react-native"
+import CustomText from "../ui/CustomText"
```
Comment (serapieTuyishime) [inline components/cards/Header.tsx:2]:
```suggestion
import CustomText from "@/components/ui/CustomText"
```

File: components/vectors/RightIcon.tsx
Comment (serapieTuyishime) [inline components/vectors/RightIcon.tsx:1]: Is this vector used?

File: screens/EditSignal/index.tsx [hunk 2]
```diff
@@ -61,18 +62,7 @@ export default function EditSignal() {
...
-        </CustomText>
-        <TouchableOpacity
-          onPress={() => {
-            navigation.goBack()
-          }}
-          style={style.CrossMarkButton}>
-          <CrossMark />
-        </TouchableOpacity>
-      </View>
+      <Header title=" Edit status" />
```
Comment (Odile078) [inline screens/EditSignal/index.tsx:65]:
```suggestion
      <Header title="Edit status" />
```

File: screens/EditSignal/index.tsx [hunk 3]
```diff
@@ -91,7 +81,13 @@ export default function EditSignal() {
           timeSlots={["NOW", "MORNING", "LUNCH", "AFTERNOON", "EVENING"]}
         />
         <FriendsList />
-        <ShareCard style={{ marginHorizontal: 20 }} />
+        <ConfirmAction
+          title=" Your friends are not on Wavv?"
```
Comment (Odile078) [inline screens/EditSignal/index.tsx:85]:
```suggestion
          title="Your friends are not on Wavv?"
```

File: screens/Search/index.tsx [hunk 4]
```diff
@@ -163,7 +155,12 @@ const FindFriends = () => {
         )}
 
         <View style={styles.share}>
-          <ShareCard />
+          <ConfirmAction
+            title=" Your friends are not on Wavv?"
```
Comment (Odile078) [inline screens/Search/index.tsx:159]:
```suggestion
            title="Your friends are not on Wavv?"
```

File: screens/Settings/index.tsx [hunk 5]
```diff
@@ -13,56 +15,30 @@ export default function EntryScreen() {
...
-          />
-        </View>
+    <SafeAreaView style={styles.container}>
+      <StatusBar style="dark" />
+      <Header title="Manage account" />
+      <View
+        style={{
+          paddingHorizontal: 20,
+          width: "100%",
+          gap: 20,
```
Comment (Odile078) [inline screens/Settings/index.tsx:25]: do we need this gap and width styles?

**Review by igorntwari — APPROVED** [review]

**Review by serapieTuyishime — APPROVED** [review]

**Pull request merged**